.gitignore

```diff
@@ -106,5 +106,6 @@ Temporary Items
 
 /states/
 /rooms/
+!/rooms/important
```

Comment:
I think you got it mixed up. You removed Test1.json. My guess is that you put it in the root of /rooms, which is what I meant in my previous comment, but then instead of `!/rooms/Test1.json`, you added this line which won't work.

Reply:
Now it should work

Reply:
But now you have to put it manually in /rooms :( If you put it in /rooms and add this specific file to .gitignore then it would be always available